src/journey.js

```diff
@@ -32,6 +33,7 @@ const genWandering = () => [0, 1].map(() => toDirection[d6()]);
 const genEncounter = () => (d20() >= 16 ? d100() : "none");
 
 const getWeather = day => assoc("weather", genWeather(), day);
+export const setSeed = seed => { seedrandom(seed, {global: true}); }
```

Comment:
Would you mind putting a comment above this line, indicating that `global: true` is going to affect the `Math` prototype? At least, I _think_ that's what is happening, based on the documentation.

Reply:
Changed this in: 66fbdb0